misc/header.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
Done, I also removed the babel files as we don't need any translation in the REST API.